easybuild/easyconfigs/v/Velvet/Velvet-1.2.10-goolf-1.4.10-mt-kmer_31.eb

```diff
@@ -23,6 +23,6 @@ source_urls = ['http://www.ebi.ac.uk/~zerbino/%(namelower)s']
 # by default MAXKMERLENGTH=31 but defined here to keep all the easyconfigs homogeneous
 buildopts = "OPENMP=1 MAXKMERLENGTH=%s" % versionsuffix.split('_')[1]
 
-postinstallcmds = ["cp -a contrib %(installdir)s/"]
+postinstallcmds = ["cd contrib/MetaVelvet-1.2.01 && make && cd ../../ && cp -a contrib %(installdir)s/"]
```

Comment:
avoid hardcoding the version here; since it's different from the Velvet version, use:

```
postinstallcmds = ["cd contrib/MetaVelvet-*/ && make && cd ../../ && cp -a contrib %(installdir)s/"]
              
```